.github/workflows/charts-lint-test-scan.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint, test, and scan Helm charts
+on:
+  pull_request:
+    branches:
+      - master
+      - release/**
+  workflow_dispatch:
+jobs:
+  lint-test-scan:
+    uses: Cray-HPE/.github/.github/workflows/charts-lint-test-scan.yml@main
+    with:
+      lint-charts: ${{ github.event_name == 'pull_request' }}
+      ct-config: |
+        chart-dirs:
+          - kubernetes
+        validate-maintainers: false
+      scan-chart-snyk-args: "--severity-threshold=high --policy-path=kubernetes/.snyk"
+      test-charts: false
+      scan-images: false
```

Comment:
If we're scanning images nightly (in the charts-lint-test-scan-cron.yml workflow), is there a particular reason why we would not also want to scan them with PRs?

Reply:
Agree, I'll make it scan with PRs as well in the next patch.